### ISSUE TITLE
Incorrect presentation role on Tooltips in CMS

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10221]
## Description
Within the read only text sections of the CMS, there is a tooltip that when hovered over or focused on displays additional information.  However, a screen reader does not read this tooltip due to the presentation role making it not available to screen reader users

## Screenshot
![image](https://user-images.githubusercontent.com/106678594/184380802-e08e9dd2-ef25-42c1-8b65-328ec8f2b2f3.png)

## Accessibility Standard

WCAG version 2.1 AA, [Criterion 1.4.13](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html)

## Acceptance Criteria
- [ ] UX/IA review
- [ ] Interactive Design review and documentation in Design System
- [ ] Technical review
- [ ] Change management consulted

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by omahane** [comment]
@laflannery Would `role="tooltip"` suffice to fix this?

**Comment by laflannery** [comment]
@omahane there would be a couple things I would want to update for this particular defect. However, I actually have a couple other a11y defects and in general some overall UX thoughts (i.e. can we present this info in a better way, are tooltips actually the best way to do this?).

Are you looking at this for fun - or are you looking to work on this? Because if it's the latter I can give you some more details

**Comment by laflannery** [comment]
Talked to Blake and we are going have tooltips go through his Design system process to look at these more holistically

**Comment by srancour** [comment]
This is something that still exists and should still be worked on. @MDomngz we should look through what the last team did with the design in #6834 and #10777 for what we might want to do here.
